tests/test_amex_browser_workflow.py

```
@@ -0,0 +1,652 @@
+        )
+
+    @patch.object(downloader.time, "sleep")
+    @patch.object(downloader.messages, "get_db")
+    def test_browserbase_reads_recent_amex_sms_code(
+        self, get_db: MagicMock, sleep: MagicMock
+    ) -> None:
+        message = MagicMock(is_from_me=False, text="AMEX: your code is 315238")
+        get_db.return_value.chats.return_value = [MagicMock(id=42)]
+        get_db.return_value.messages.return_value = [message]
+
+        code = downloader.wait_for_amex_sms_code(datetime.now())
```

Ruff (0.16.1)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)